pkgs/tools/system/fancy-motd/default.nix

```diff
@@ -1,21 +1,21 @@
-{ stdenv, lib, fetchFromGitHub, bc, curl, figlet, fortune, gawk, iproute2, procps }:
+{ stdenv, lib, fetchFromGitHub, bc, curl, figlet, fortune, gawk, iproute2, lm_sensors, procps }:
 
 stdenv.mkDerivation rec {
   pname = "fancy-motd";
-  version = "unstable-2022-06-06";
+  version = "unstable-2022-11-08";
```

Comment:
Could you adapt the new version scheme for derivations based on unreleased sources? See https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming